### PR TITLE
rename get_workspace_status to get_workspace_status.sh in /bin

### PR DESCRIPTION
**Pull request by m1093782566**
rename get_workspace_status to get_workspace_status.sh in /bin

### CODE REVIEW

**Comment by m1093782566** [comment]
@ymesika 

> gobuild.sh which ref this one should be updated accordingly.

Thanks for your comments, it's fixed now. PTAL.

**Comment by douglas-reid** [comment]
lint failure: `bin/gobuild.sh: line 56: /go/src/istio.io/istio/bin/get_workspace_status.sh: Permission denied`

File: bin/gobuild.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ fi
 # at the beginning of the build and used throughout
 if [[ -z ${BUILDINFO} ]];then
```
Comment (sdake) [inline bin/gobuild.sh]: please undo the mode change to 644.  Executables should be 755.

Cheers
-steve

Comment (m1093782566) [reply bin/gobuild.sh]: OOPS... I developed it on Windows platform and the file mode was changed automatically.

**Comment by m1093782566** [comment]
@sdake @douglas-reid 

Comments are fixed now. Could you please approve this PR?

**Comment by m1093782566** [comment]
@ymesika Could you approve this PR now?

**Comment by ymesika** [comment]
I can but it won't progress you much because I'm not owner

**Comment by m1093782566** [comment]
@ymesika 

I know, but you can approve the changes you requested before - in github way :)

**Pull request merged**